portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/rollingrestart/RollingRestartEE.testcase

```diff
@@ -908,12 +897,6 @@ definition {
 			deleteLiferayHome = "false",
 			nodePort = "9080");
 
-		User.firstLoginUI(
```

Comment:
@Kvale1733 If you check line 891, we update the portal properties here to change our login authentication method. If I told you to remove the logout, then that is my bad, we need to relogin here in order to confirm that the property change worked correctly for cluster with port 9080.